sounds.js

```diff
@@ -35,20 +35,24 @@ function playIncorrectGuessSound() {
 
 function playNewPuzzleSound() {
   if (!newPuzzleSound.isPlaying())
+    newPuzzleSound.setVolume(0.1);
```

Comment:
you need to wrap these 2 statements inside curly braces to behave the way you intendd